File: plugins/kotlin/j2k/shared/tests/testData/newJ2k/methodCallExpression/stringConcatFormatting.java
class A {
    public static void main(String[] args) {
        String s = "";

        String result = s
          .replace("_", "/")
          .concat("==");

        String result2 = s
          .replace("_", "/") // comment
          .concat("==");

        String result3 = s
          .replace("_", "/").concat("==");

        String result4 = s
          .replace("_", "/").concat("==").replace("_", "/");

        String result5 = s
          .replace("_", "/")
          .concat("==")
          .replace("_", "/");

        String result6 = s
          .replace("_", "/")
          .concat("==") // comment
          .replace("_", "/");
    }
}

File: plugins/kotlin/j2k/new/tests/testData/newJ2k/methodCallExpression/stringConcatFormatting.kt
internal object A {
    @JvmStatic
    fun main(args: Array<String>) {
        val s = ""

        val result = s
            .replace("_", "/") + "=="

        val result2 = (s
            .replace("_", "/") // comment
                + "==")

        val result3 = s
            .replace("_", "/") + "=="

        val result4 = (s
            .replace("_", "/") + "==").replace("_", "/")

        val result5 = (s
            .replace("_", "/") + "==")
            .replace("_", "/")

        val result6 = (s
            .replace("_", "/") + "==") // comment
            .replace("_", "/")
    }
}
